vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -963,6 +963,36 @@ testNBCParserBinary () {
 
 }
 
+testWasmRuntimesInstalled () {
+  local test="testWasmRuntimesInstalled"
+  local wasm_runtimes_path="/usr/local/bin"
+  local spin_runtime_versions="v0.3.0 v0.5.1"
```

Comment:
Is it possible to avoid hardcoding these?

Reply:
well, I don't see much value from avoiding hardcoding these. I took a look at other tests and they seem to have hardcoded paths too.

Reply:
Concern is that every time we need an update or if these versions are removed from publishing or code, we need to update these test files too, or they'll fail

Reply:
We even had a discussion about issues related to such hardcoding today. cc: @Devinwong, @AlisonB319, are we ok with hardcoding these here? Will later refactors to consolidate to components.json and to use ORAS be affected.

Reply:
Yeah I think it is fine to hard code for now, I'm currently working on a PR to update WASM so I'll take care of making sure they don't stay hard coded for very much longer :)